mods/voidmulti.sc2mod/base.sc2data/gamedata/unitdata.xml

```diff
@@ -1182,7 +1360,7 @@
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowInfestorUp,Execute" Row="2" Column="4"/>
             <LayoutButtons Face="BurrowUp" Type="AbilCmd" AbilCmd="BurrowUltraliskUp,Execute" Row="2" Column="4"/>
         </CardLayouts>
-        <BehaviorArray Link="QueenMustBeOnCreep"/>
+        <BehaviorArray Link="OnCreep"/>
```

Comment:
**undocumented**, **broken**: This breaks the text displayed on the button while the Queen is not on Creep

The patch notes do not indicate any Transfusion change. So this should be reverted to QueenMustBeOnCreep!